src/IO/WriteHelpers.h

```diff
@@ -798,6 +798,28 @@ inline void writeDateTimeTextRFC1123(time_t datetime, WriteBuffer & buf, const D
     buf.write(" GMT", 4);
 }
 
+inline void writeText(const String & x, WriteBuffer & buf) { writeEscapedString(x, buf); }
+
+inline void writeDateTimeTextUnixTimestamp(time_t datetime, WriteBuffer & buf, const DateLUTImpl & date_lut = DateLUT::instance())
+{
+    const auto & values = date_lut.getValues(datetime);
+    auto local_date_time_unix_timestamp = static_cast<time_t>(LocalDateTime(values.year, values.month, 
+        values.day_of_month, date_lut.toHour(datetime), date_lut.toMinute(datetime), date_lut.toSecond(datetime)));
+    writeText(std::to_string(local_date_time_unix_timestamp), buf);
+}
+
+inline void
+writeDateTimeTextUnixTimestamp(DateTime64 datetime64, UInt32 scale, WriteBuffer & buf, const DateLUTImpl & date_lut = DateLUT::instance())
```

Comment:
Here you should write unix_timestamp.fractional_part.

Reply:
Currently toUnixTimestamp function if applied to DateTime64 column does not write fractional_part in addition to unix_timestamp. Should we write that ?
```
CREATE TABLE test_date_time_64(timestamp DateTime64) ENGINE=Log;
INSERT INTO test_date_time_64 VALUES (1546300800000);
SELECT toUnixTimestamp(timestamp) FROM test_date_time_64;


┌─toUnixTimestamp(timestamp)─┐
│                 1546300800 │
└────────────────────────────┘
```

Reply:
It's in question how this function should work. Now it does not return fractional part, it's Ok (it returns unix timestamp, indeed).